src/elexmodel/handlers/data/ModelResults.py

```diff
@@ -115,6 +122,8 @@ def write_data(self, election_id, office, geographic_unit_type):
             self.process_final_results()
         s3_client = s3.S3CsvUtil(TARGET_BUCKET)
         for key, value in self.final_results.items():
+            if keys is not None and key not in keys:
```

Comment:
If you pass in `keys` like `keys=["nat_sum_data"]`, then only the `nat_sum_data/current.csv` will be written.  It's to prevent every single CSV being written every single time this method is called, since we only want `nat_sum_data/current.csv` to be updated when `get_national_summary_votes_estimates()` is called 🤔